src/auth.py

```diff
@@ -157,7 +190,7 @@ def add_user(self, username: str, password: str) -> None:
         ]
         KafkaSnap.run_bin_command(bin_keyword="configs", bin_args=command, opts=self.opts)
 
-    def delete_user(self, username: str) -> None:
+    def delete_user(self, username: str, **_) -> None:
```

Comment:
**question** looks weird this second argument. what is this for?

Reply:
Ah good catch. It was a leftover from a first implementation to allow for us to pass arbitrary keyword arguments from a dict, without them throwing errors. We don't 'need' it now though so I will remove, but for context we're using the pattern already here: 

https://github.com/canonical/kafka-operator/blob/58fbfcdd50c8ffe3daae9dedd0f9ce3e3baa69bf/src/auth.py#L247-L260